src/test/java/org/apache/commons/lang3/time/FastDatePrinterTest.java

```diff
@@ -432,4 +432,15 @@ public void testDayNumberOfWeek() {
         calendar.set(Calendar.DAY_OF_WEEK, Calendar.SUNDAY);
         assertEquals("7", printer.format(calendar.getTime()));
     }
+
+    @DefaultLocale(language = "en", country = "US")
+    @DefaultTimeZone("America/New_York")
+    @Test
+    public void testWeekYear() {
+        final GregorianCalendar cal = new GregorianCalendar(2020, 12, 31, 0, 0, 0);
+        final DatePrinter printer4Digits = getInstance("YYYY");
+        final DatePrinter printer2Digits = getInstance("YY");
```

Comment:
What about "YYY"?

Reply:
Please add code in your new test for Y and YYY.

Reply:
added, thanks